src/cdk/explorer/nodes/getCfnDefinition.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+
+/**
+ * @param uniqueIdentifier unique identifier of state machine
+ * @param cdkOutPath cdk.out path
```

Comment:
Given that this functionality is something that we will likely reuse for non-CDK related functionality, it likely makes sense for at least some of this code to live in the Step Functions module. I think we should generalize this to be `templatePath`, and have the caller provide the path to the template. Take a look through the code in this file and see if we can identify which of it is specific to the CDK, and which of it is general to all CloudFormation templates, then move that code to the appropriate module.